tests/python/contrib/test_hexagon/test_maxpool2d_blocked.py

```diff
@@ -151,7 +151,7 @@ def test_maxpool(self, shape_nhwc, window_size, stride, pad, dtype, target):
             padding=(pad, pad, pad, pad),
             dtype=dtype,
         )
-        return output, ref_output
+        assert all([output is not None, ref_output is not None])
```

Comment:
If the assert is causing spurious unit test failures, it can be removed.  The test is to ensure that no exceptions are through while lowering/building, and no further assert is required.

Reply:
I did remove it at first, but then the linter complained that `output` and `ref_output` were unused values...  :|